app/src/main/java/no/usn/mob3000/ui/screens/info/faq/FAQScreen.kt

```diff
@@ -88,7 +90,8 @@ fun FAQScreen(
         floatingActionButton = {
             FloatingActionButton(
                 onClick = onCreateFAQClick,
-                containerColor = DefaultButton
+                containerColor = MaterialTheme.colorScheme.primary,
+                modifier = Modifier.shadow(15.dp)
```

Comment:
[See this comment](https://github.com/frigvid/mob3000-group11/pull/19/files#r1840963289).